backend/Squirrel.AzureBlobStorage/Interfaces/IBlobStorageService.cs

```diff
@@ -4,8 +4,8 @@ namespace Squirrel.AzureBlobStorage.Interfaces;
 
 public interface IBlobStorageService
 {
-    Task<Blob> UploadAsync(string containerName, Blob blob);
-    Task<Blob> UpdateAsync(string containerName, Blob blob);
+    Task UploadAsync(string containerName, Blob blob);
```

Comment:
probably we should return URL or ID/GUID?

Reply:
No we dont need to return this